pkg/combustion/templates/rke2-multi-node-installer.sh.tpl

```diff
@@ -91,10 +91,9 @@ systemctl enable kubernetes-resources-install.service
 {{- end }}
 fi
 
-{{- if .apiHost }}
+{{- if and .apiVIP .apiHost }}
```

Comment:
I don't necessarily see an issue with this but if we do it, let's do it in all other kubernetes templates too.

Reply:
It's already present in all of the other templates, it was just missing from this one for some reason